lib/straight/exchange_rate_adapters/average_rate_adapter.rb

```diff
@@ -0,0 +1,22 @@
+module Straight
+  module ExchangeRate
+
+    class AverageRateAdapter
```

Comment:
Now, it may seem like you don't need to inherit from Adapter, but in fact it is a good idea to do so. You can reload all the methods you don't need and raise an exception inside them saying "This method is not supposed to be used in `AverageRateAdapater`.